nntrainer/graph/network_graph.cpp

```diff
@@ -779,17 +779,21 @@ NetworkGraph::finalizeContext(const std::shared_ptr<LayerNode> &lnode,
           TensorSpecV2::RequestType::READ_ONLY_VIEW;
         if (lnode->getType() == IdentityLayer::type) {
           s.variable_spec.reference_name = inputs[i]->getName();
+          s.variable_spec.dim.setFormat(inputs[i]->getDim().getFormat());
```

Comment:
how about using `getFormat()` directly instead of calling `getDim()`?
```suggestion
          s.variable_spec.dim.setFormat(inputs[i]->getFormat());
```
using `Tensor::getFormat()` could also have a memory advantage since `getDim()` returns a copy of the TensorDim.